npm/extensions.bzl

```diff
@@ -221,7 +221,7 @@ def _npm_translate_lock_attrs():
     attrs["run_lifecycle_hooks"] = attr.bool(default = True)
 
     # Args defaulted differently by the macro
-    attrs["npm_package_target_name"] = attr.string(default = "{dirname}")
```

Comment:
Lets just make sure the `{dirname}` var is used in a test somewhere still.

Reply:
Already used in `rules_js/e2e/npm_link_package`:

```
npm_translate_lock(
    name = "npm",
    lifecycle_hooks_envs = {
        "*": [
            "aspect_test_id=testc",
            "BUILD_VERSION=1",
        ],
        # Without APPDATA environment variable, sharp install will fail because of the binary can not
        # be copied to ~/.npm folder which is readonly for bazel.
        "sharp": ["APPDATA=."],
    },
    lifecycle_hooks_execution_requirements = {
        "sharp": ["requires-network"],
    },
    npm_package_target_name = "{dirname}_pkg",
    pnpm_lock = "//:pnpm-lock.yaml",
    verify_node_modules_ignored = "//:.bazelignore",
)
```